src/compute/src/logging/compute.rs

```diff
@@ -347,16 +347,23 @@ pub fn construct<A: Allocate>(
 
         let frontier_delay = frontier_delay
             .as_collection()
+            .explode_one(|(dataflow, source_id, worker, delay)| {
+                (
+                    (dataflow, source_id, worker, delay.next_power_of_two()),
+                    (i128::try_from(delay).expect("delay too big"), 1),
+                )
+            })
             // TODO(#16549): Use explicit arrangement
             .count_total_core::<i64>()
             .map({
-                move |((dataflow, source_id, worker, delay_pow), count)| {
+                move |((dataflow, source_id, worker, delay_pow), (sum, count))| {
                     Row::pack_slice(&[
                         Datum::String(&dataflow.to_string()),
                         Datum::String(&source_id.to_string()),
                         Datum::UInt64(u64::cast_from(worker)),
                         Datum::UInt64(delay_pow.try_into().expect("pow too big")),
-                        Datum::Int64(count.into()),
+                        Datum::Int64(count.try_into().expect("count too big")),
```

Comment:
Why did this have to change?

Reply:
Because `count` is now an `i128`, which is required by the `Multiply` type bound on `explode_one`. I'm not sure if it will still be required when I implement your suggestion to remove `explode_one`; I will check.